client/src/i18n/en/debtor_group.json

```diff
@@ -32,4 +32,8 @@
 "SUBSCRIBED":"Debtors subscribed",
 "UPDATED":"Debtor group record updated",
 "LOADING":"Fetching Debtor Group",
+"CONVENTION":{
+  "TITLE":"Convention",
+  "DESCRIPTION":"A convention group is a group (enterprise) which takes charge of the invoices of the patients belonging to this group"
```

Comment:
A better English translation for this might be:
> The invoices of debtors in this group will be covered by this debtor group's account.  Patients belonging to this group will be blocked from paying for their bills at the cash window.